cmd/whereabouts.go

```diff
@@ -76,7 +76,12 @@ func cmdAdd(args *skel.CmdArgs, client *kubernetes.KubernetesIPAM, cniVersion st
 	ctx, cancel := context.WithTimeout(context.Background(), types.AddTimeLimit)
 	defer cancel()
 
-	newips, err := kubernetes.IPManagement(ctx, types.Allocate, client.Config, client)
+	if client.Config.Range != "" && client.Config.NodeSliceFastRange != "" {
```

Comment:
I guess error handling mixed configurations will depend on how we want to define the new IPAM Config fields